server/reload.go

```diff
@@ -280,6 +281,9 @@ func (c *clusterOption) Apply(server *Server) {
 	server.setRouteInfoHostPortAndIP()
 	server.mu.Unlock()
 	server.Noticef("Reloaded: cluster")
+	if c.warnTLSInsecure {
```

Comment:
Will this ever be false?

Reply:
Yes. This is set to true only if "insecure" property moves from false to true. That is, if there was no TLS config, or TLS config's insecure was false and now is true, warn, otherwise, do not.
Again, I could make it simpler and always warn if the property is true (even if it was already true and already got the warning).